connector/extensions/report-service/src/main/java/org/oaebudt/edc/report/model/ReportType.java

```diff
@@ -1,5 +1,5 @@
 package org.oaebudt.edc.report.model;
 
 public enum ReportType {
-    TITLE_REPORT, ITEM_REPORT
+    TITLE_REPORT, ITEM_REPORT, TEST_REPORT
```

Comment:
what is this `TEST_REPORT`?

An advice for the future, better not to use enum for representing constants, because they are forcing to have a code change + deploy in case of extension needed, e.g. if they would like to add a new type of report at the moment won't be possible without changing the code

Reply:
Test Report was added when I was running some tests. I should actually remove it.
I understand that we need to keep these in a store and provide a way for providers to configure it. (I had thought about that)

But this was already added and it was stated that we consider this report for the POC. 

This will be made configurable for the MVP  stage.